tensorflow/python/ops/losses/losses_impl.py

```diff
@@ -739,7 +739,7 @@ def sigmoid_cross_entropy(
 
 @tf_export("losses.softmax_cross_entropy")
 def softmax_cross_entropy(
-    onehot_labels, logits, weights=1.0, label_smoothing=0, scope=None,
+    onehot_labels, logits, axis=-1, weights=1.0, label_smoothing=0, scope=None,
```

Comment:
In general it is not backward compatible to add new arguments anywhere except the end. If you think this is a better position for this argument, you can create two versions. A v1 version with it added at the end, a v2 version with it in this position, and a rule in tensorflow/tools/compatibility/tf_upgrade_v2.py.

Reply:
Done please review the proposed changed.